### PR TITLE
Phishing labels 2

### PR DESCRIPTION
**Pull request by tomer-pan**
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/43049 https://github.com/demisto/etc/issues/42839

## Description
1) Replacing incident labels with incident fields - This will help with generalizing the playbooks to work with other integrations as well (MSGraph).
2) Extracting indicators on incident creation - will extract indicators from email body for example
Forwarded emails:
    - New fields for differentiating attacker and reporter details. 1 New field for getting the origin of the processed mail. 
    - Better visibility in layout (v1) - new fields displayed, no more confusion when dealing with forwarded email.
    - Retrieving eml instead of email objects (EWS + MSGraph) - this allows a better process of the emails.
3) URL Detonation playbook has been added.
4) New playbooks:
    - Phishing - Generic (v3)
    - Get Original Email - Gmail (v2), EWS (v2), MSGraph
    - Search And Delete - Gmail
    - Process Email - Generic V2
5) Some other minor fixes and improvements like adding new/missing inputs and typos fix.

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details: Changing labels to fields will break the flow if the customer is not using our incoming mappers. That's why we have created v2/v3 for these playbooks.
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by julieschwartz18** [comment]
@tomer-pan - is this PR ready for doc review?

**Comment by tomer-pan** [comment]
@julieschwartz18 It is now, thanks!
I will update in case I will need to add anything.
@David-BMS FYI

**Comment by julieschwartz18** [comment]
@tomer-pan - playbook-Get_Original_Email_-_Generic_v2_README.md looks fine, just 2 things:
1. MessgaeID playbook input name is misspelled - it should be corrected (in the yml and in the README) 
2. Missing the link for the playbook image

**Comment by julieschwartz18** [comment]
@tomer-pan - doc review is complete for this PR. 
Please regenerate these files:
1. playbook-Get_Original_Email_-_EWS_v2_README.md 
2. Phishing_-_Generic_v3_README.md 
Let me know when you do this and I will then select docs-approved. Thanks!

**Comment by tomer-pan** [comment]
> @tomer-pan - doc review is complete for this PR. Please regenerate these files:
> 
> 1. playbook-Get_Original_Email_-_EWS_v2_README.md
> 2. Phishing_-_Generic_v3_README.md
>    Let me know when you do this and I will then select docs-approved. Thanks!

I'm done!
After your approval, will be doing some further tests to verify all is working properly.
Thanks!

File: Packs/Phishing/IncidentFields/incidentfield-reported_email_origin.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "associatedTypes": [
+        "Phishing"
+    ],
+    "associatedToAll": false,
+    "unmapped": false,
+    "unsearchable": true,
+    "caseInsensitive": true,
+    "sla": 0,
+    "threshold": 72,
+    "fromVersion": "5.5.0"
```
Comment (David-BMS) [inline Packs/Phishing/IncidentFields/incidentfield-reported_email_origin.json:38]: If new field change to 6.0

File: Packs/Phishing/Playbooks/Phishing_-_Generic_v3_README.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
+This playbook investigates and remediates a potential phishing incident. It engages with the user that triggered the incident while investigating the incident itself.
+Note: Final remediation tasks are always decided by a human analyst.
```
Comment (David-BMS) [inline Packs/Phishing/Playbooks/Phishing_-_Generic_v3_README.md:2]: Missing a reference to that this is for 6.1 and that we are using indicator extraction rules

Comment (tomer-pan) [reply Packs/Phishing/Playbooks/Phishing_-_Generic_v3_README.md:2]: Done. @julieschwartz18 can you please take a look here?
https://github.com/demisto/content/pull/16327/commits/b0328c57e0839615a6fbfd15ac2e817f6cc821ae

File: Packs/Phishing/IncidentFields/incidentfield-reported_email_origin.json (hunk 1 above)
Comment (tomer-pan) [reply Packs/Phishing/IncidentFields/incidentfield-reported_email_origin.json]: Done

File: Packs/Phishing/Playbooks/Phishing_-_Generic_v3_README.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+| InternalRange | This input is used in the task "Entity Enrichment - Phishing v2" playbook.<br/>A list of internal IP ranges to check IP addresses against. The list should be provided in CIDR notation, separated by commas. An example of a list of ranges is: "172.16.0.0/12,10.0.0.0/8,192.168.0.0/16" \(without quotes\). If a list is not provided, uses the default list provided in the IsIPInRanges script \(the known IPv4 private address ranges\). |  | Optional |
+| PhishingModelName | Optional - the name of a pre-trained phishing model to predict phishing type using machine learning. | phishing_model | Optional |
+
+## Playbook Outputs
+---
+There are no outputs for this playbook.
+
+## Playbook Image
+---
+![Phishing - Generic v3](Insert the link to your image here)
```
Comment (David-BMS) [inline Packs/Phishing/Playbooks/Phishing_-_Generic_v3_README.md:104]: Missing image

File: Packs/Phishing/Playbooks/Process_Email_-_Generic_README.md [hunk 4]
```diff
@@ -1,56 +1,58 @@
...
 | Email.Headers | The full email headers as a single string. | string |
 | Email.Attachments | The list of attachment names in the email. | string |
-| Email.Format | The format of the email if available. | string |
+| Email.Format | The email format if available. | string |
 | File | The file object. | unknown |
 
 ## Playbook Image
 ---
-![Process_Email_Generic](https://raw.githubusercontent.com/demisto/content/82895af983e287954ef4565db548f9ae91d0487a/Packs/Phishing/doc_files/Process_Email_-_Generic.png)
+![Process Email - Generic](https://raw.githubusercontent.com/demisto/content/82895af983e287954ef4565db548f9ae91d0487a/Packs/Phishing/doc_files/Process_Email_-_Generic.png)
```
Comment (David-BMS) [inline Packs/Phishing/Playbooks/Process_Email_-_Generic_README.md:58]: You can also more to relative path.

File: Packs/Phishing/Playbooks/Process_Email_-_Generic_v2_README.md [hunk 5]
```diff
@@ -72,4 +72,4 @@ This playbook does not use any integrations.
 
 ## Playbook Image
 ---
-![Process Email - Generic v2](https://raw.githubusercontent.com/demisto/content/07a19d09dad3bfef74e03552446107a973752fe2/Packs/Phishing/doc_files/Process_Email_-_Generic_v2.png)
+![Process Email - Generic v2](https://raw.githubusercontent.com/demisto/content/07a19d09dad3bfef74e03552446107a973752fe2/Packs/Phishing/doc_files/Process_Email_-_Generic_v2.png)
```
Comment (David-BMS) [inline Packs/Phishing/Playbooks/Process_Email_-_Generic_v2_README.md:75]: Change to relative path

File: Packs/Phishing/Playbooks/Process_Microsoft's_Anti-Spam_Headers.yml [hunk 6]
```diff
@@ -1343,37 +1343,37 @@ inputs:
...
   description: |-
-    Set the minimum severity of an incident with a BCL (bulk sender) that has a score equal to or higher than 4.
+    Sets the minimum severity of an incident with a Bulk Complaint Level (BCL) score equal to or greater than 4.
 
-    Available values: 0 (Unknown), 1 (Low), 2 (Medium), 3 (High), 4 (Critical).
+    Possible values: 0 (Unknown), 1 (Low), 2 (Medium), 3 (High), 4 (Critical).
   playbookInputQuery:
 - key: PCL-Severity
   value:
     simple: "2"
```
Comment (David-BMS) [inline Packs/Phishing/Playbooks/Process_Microsoft's_Anti-Spam_Headers.yml:1352]: This is the intended default value?

File: Packs/Phishing/Playbooks/Process_Microsoft's_Anti-Spam_Headers_README.md [hunk 7]
```diff
@@ -34,16 +28,16 @@ This playbook does not use any integrations.
...
 
 ## Playbook Outputs
 ---
 
 | **Path** | **Description** | **Type** |
 | --- | --- | --- |
-| Email.MicrosoftHeadersSeverityCheck | Possible Values:<br/><br/>Medium: PCL or BCL scores are equal to or higher than 4.<br/><br/>High: BCL score is equal to or higher than 8.<br/> | unknown |
+| Email.MicrosoftHeadersSeverityCheck | Possible Values:<br/><br/>Medium: PCL or BCL scores are equal to or greater than 4.<br/><br/>High: BCL score is equal to or greater than 8.<br/> | unknown |
 
 ## Playbook Image
```
Comment (David-BMS) [inline Packs/Phishing/Playbooks/Process_Microsoft's_Anti-Spam_Headers_README.md:42]: missing image

File: Packs/Phishing/README.md [hunk 8]
```diff
@@ -1,24 +1,27 @@
...
+- Engages with the end user regarding the incident such as notifying them of receipt of email and providing further instructions if an email is found to be malicious.
 
 As part of this pack, you will also get out-of-the-box phishing incident views, a full layout and automation scripts. All of these are easily customizable to suit the needs of your organization.
 
 _For more information, visit our [Cortex XSOAR Developer Docs](https://xsoar.pan.dev/docs/reference/playbooks/phishing-investigation---generic-v2)._
 
 _If you're new to our Phishing Content Pack, check out our [Setting Up a Phishing Incident in Cortex XSOAR](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-2/cortex-xsoar-tutorials/tutorials/phishing-use-case.html) tutorial._
 
-![Phishing_Investigation_Generic_v2](https://raw.githubusercontent.com/demisto/content/952153821f8de373a5d527df653bfa845d32f77e/Packs/Phishing/doc_files/Phishing_Investigation_-_Generic_v2_-_6_0.png)
+![Phishing_Investigation_Generic_v2](https://raw.githubusercontent.com/demisto/content/13bc88b052c5039775cd0f569259e7fd0e1b7dea/Packs/Phishing/doc_files/Phishing_-_Generic_v3.png)
```
Comment (David-BMS) [inline Packs/Phishing/README.md:27]: Change to relative path

File: Packs/Phishing/README.md [hunk 9]
```diff
@@ -1,24 +1,27 @@
...
+- Runs checks for SSL certificates of URLs, email address breach involvement, domain-squatting and email authenticity using SPF, DKIM and DMARC checks.
+- Identifies similar phishing incidents belonging to the same campaign, providing visibility and manual or automatic actions to respond to such incidents.
+- Calculates severity for the incident based on the provided initial severity, indicator reputations, email authenticity check, and critical assets if any are involved.
+- Remediates the incident by blocking malicious indicators, searching for and deleting malicious emails upon analyst approval.
+- Engages with the end user regarding the incident such as notifying them of receipt of email and providing further instructions if an email is found to be malicious.
 
 As part of this pack, you will also get out-of-the-box phishing incident views, a full layout and automation scripts. All of these are easily customizable to suit the needs of your organization.
 
 _For more information, visit our [Cortex XSOAR Developer Docs](https://xsoar.pan.dev/docs/reference/playbooks/phishing-investigation---generic-v2)._
```
Comment (David-BMS) [inline Packs/Phishing/README.md:23]: Make sure to use the updated articles links (not v2 etc)

File: Packs/Gmail/Playbooks/playbook-Search_And_Delete_Emails_-_Gmail.yml [hunk 10]
```diff
@@ -1,191 +1,164 @@
-description: This playbook searches Gmail to identify and delete emails with similar
```
Comment (David-BMS) [inline Packs/Gmail/Playbooks/playbook-Search_And_Delete_Emails_-_Gmail.yml:1]: Weren't we suppose to provide a disclaimer here about the google search limitations?

**Review by David-BMS — COMMENTED** [review]
Please review provided comments

File: Packs/Phishing/Playbooks/Process_Microsoft's_Anti-Spam_Headers.yml (hunk 6 above)
Comment (tomer-pan) [reply Packs/Phishing/Playbooks/Process_Microsoft's_Anti-Spam_Headers.yml:1352]: Yes - this is the minimum severity set for an incident when the score is higher then 4 (it is written in the description - we went through this when I was building this playbook).

File: Packs/Phishing/README.md (hunk 8 above)
Comment (tomer-pan) [reply Packs/Phishing/README.md:27]: I get an error in the CI validations process when using a relative path in the pack README file.

File: Packs/Gmail/Playbooks/playbook-Search_And_Delete_Emails_-_Gmail.yml (hunk 10 above)
Comment (tomer-pan) [reply Packs/Gmail/Playbooks/playbook-Search_And_Delete_Emails_-_Gmail.yml:1]: Added one.
@julieschwartz18 Can you please review: https://github.com/demisto/content/pull/16327/commits/b3a85589394f9d30c9f638431d025d07d9ffb03b

File: Packs/Phishing/README.md (hunk 9 above)
Comment (tomer-pan) [reply Packs/Phishing/README.md:23]: Once the new article will be online I will set the link to the new article.

**Review by David-BMS — APPROVED** [review]

**Pull request merged**